app/src/renderer/vuex/modules/wallet.js

```diff
@@ -30,25 +31,25 @@ export default ({ node }) => {
       Vue.set(state.balances, findBalanceIndex, balance)
     },
     setWalletAddress(state, address) {
-      Vue.set(state, `address`, address)
+      state.address = address
     },
     setAccountNumber(state, accountNumber) {
```

Comment:
i know this isn't your choice @fedekunze but does anyone know what an accountNumber is? lol that's not a thing we're supposed to have.

Reply:
an `accountNumber` is a method to prevent pruned to accounts be claimed by someone else. When you create an account for the same address another time, the accountNumber increases. This comes from the SDK.